docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+services:
+  backend:
+    image: node:18-alpine3.15
+    restart: "always"
+    volumes:
+      - ./backend:/home/node
+    ports:
+      - 3001:3000
+    working_dir: /home/node
+    command: npm run start:dev
+    depends_on:
+      - db
+  frontend:
+    image: node:18-alpine3.15
+    restart: "always"
+    volumes:
+      - ./frontend:/home/node
+    ports:
+      - 3000:3000
+    working_dir: /home/node
+    command: npm run dev
+    environment:
+      - WATCHPACK_POLLING=true 
+  db:
+    image: postgres:15
+    restart: "always"
+    volumes: 
+      - postgres-volume:/var/lib/pgsql/data
+      - ./db/setup:/docker-entrypoint-initdb.d/
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PW}
+      - POSTGRES_DB=${DB_DB}
+    healthcheck:
+      # TODO: This can be replaced by a more accurate healthcheck based on PID 1 
+      test: ["CMD_SHELL","pg_isready"]
```

Comment:
![image](https://user-images.githubusercontent.com/64607316/200198143-7a27f6f3-3235-4268-a79a-13c7a3a3575c.png)

Did this happen to anyone else?
Maybe its some versioning issue on my side?
with dash it works fine tho

Reply:
I could not recreate the issue, however adding the dash did not break anything for me so I added it.
This may have been a backwards compatibility issue. Please try again and verify that it works now.